templates/manage_certs.py

```diff
@@ -32,7 +33,7 @@ def get_all_domains(config):
             line = line.strip()
             if line and not line.startswith("#"):
                 domain, backend = line.split(None, 1)
-                domains[domain] = backend
+                domains[domain.lower()] = backend
```

Comment:
@smarnach I only meant to add a unit test in that case - although an integration test is definitely a plus.

Oh, wait - where are the unit tests for this python code? I only see integration tests for the playbook as a whole in https://github.com/open-craft/ansible-load-balancer/tree/master/tests